# ESP - Geophysical Fluid Dynamics

## Villanueva, Lloyd

## Exercise I: Static

### 1. What is the head (vertical height) for water of density $1000 \, [kg/m^3]$ subject to a pressure of $500000 \, [Pa]$ ($500 \, [kPa]$)?

The height can be easily calculated using the pressure formula 
\begin{equation}
    P = \rho g h.
\end{equation}
The vertical height is calculated as
\begin{equation}
    h = \frac{P}{\rho g}
\end{equation}

In [1]:
P = 500000
rho = 1000
g = 9.81

h = P/(rho*g)
print('h =', h)

h = 50.9683995922528


Therefore, the head is about <mark>$50.97 \, [m]$</mark>

### 2. What would be the (a) gage pressure and (b) absolute pressure of water at depth $12\,[m]$ below the surface? $\rho_{water}$ = $1000\,[Kg/m^3]$ and $P_{atm} = 101000 \,[N/m^2]$ ($101\, [kN/m^2]$)

The gauge pressure is calculated as
\begin{equation}
    P_{gauge} = \rho g h
\end{equation}
and the formula for absolute pressure is 
\begin{equation}
    P_{abs} = P_{atm} + \rho g h.
\end{equation}

In [3]:
P_atm = 101000
rho_water = 1000
h = 12
g = 9.81

P_gauge = rho_water*g*h
print('Pgauge = ', P_gauge)
P_abs = P_atm + P_gauge
print('Pabs = ', P_abs)

Pgauge =  117720.0
Pabs =  218720.0


The calculated gauge pressure is <mark> $P_{gauge} = 117720.0 [Pa]$ </mark>, while the computed absolute pressure is <mark>$P_{abs} = 218720.0 [Pa]$ </mark>.

### 3. The string of a helium balloon slips from your hand and rises up into the air. When this happens:
<ol style="list-style-type: lower-alpha">
<li>The balloon rises indefinitely because helium is lighter than air.</li>
<li> <mark>  The balloon rises until it reaches an altitude where the air and helium specific weight are equal. </mark> </li>
<li> The balloon rises until it reaches an altitude where its weight equals that of the same volume of air. </li>
</ol>

##### Answer: b. The balloon rises until it reaches an altitude where the air and helium specific weight are equal.

### 4. A metal block is suspended with a wire in a tank full of cold water. The cold water is replaced by hot water with a lower specific weight. What happens to the force in the wire?
<ol style="list-style-type: lower-alpha">
    <li> <mark> Increases </mark> </li>
<li>Decreases.</li>
<li>Stays the same.</li>
</ol>

##### Answer: a. Increases

### 5.  You are standing at the bottom of a lake with your torso above water. Which statement is correct:
<ol style="list-style-type: lower-alpha">
<li>There is no buoyancy force on you since you are supported by the bottom of the lake.</li>
<li>There is a buoyancy force that is proportional to the weight of your body below the water level.</li>
<li> <mark>  There is a buoyancy force that is proportional to the volume of your body below the water level.  </mark> </li>
<li>There is a buoyancy force only when you jump up from the bottom of the lake.</li>
</ol>

##### Answer: c. There is a buoyancy force that is proportional to the volume of your body below the water level.

### 6. If liquid pressure were the same at all depths, would there be a buoyant force on an object submerged in the liquid? Explain with an example.

To answer this question, first we first look on how the the buoyant force is derived. 

First imagine a can fully submerged in water. Recall from the gauge pressure formula, that as you go deeper the pressure increases ($P_{gauge} = \rho g h$). Because of this fact the fully submerged can will experience a greater pressure in the bottom part compare to the top part. Hence, this difference pressure of the top part and bottom part create a net upward force that we will call the buoyant force. Below is how we derive the buoyant force from the pressure gradient 
\begin{align}
    F_{buoyant} & = F_{up} - F_{down} \quad \quad \text{Recall pressure is define as } P = F/A \, -> F = PA \\
    F_{buoyant} & = P_{bot} A - P_{top} A \quad \quad \text{For the sake of discussion, we let } A_{top} = A_{bot} = A \\
    F_{buoyant} & = \rho g h_{bot} A - \rho g h_{top} A \quad \quad \text{Recall the gauge pressure formula} P_{gauge} = \rho g h \\
    F_{buoyant} & = \rho g A (h_{bot} - h_{top}) \quad \quad \text{The height difference of top and bottom is just the height of the can} \\
    F_{buoyant} & = \rho g A h_{can} \\
    F_{buoyant} & = \rho g V_{can} = \rho g V_{disp} \quad \quad \text{only true when the object is fully submerge}
\end{align}
But is the volume $V$ should be only be the volume displace $V_{disp}$ by the object? 

Yes, that is right and when the object is fully submerged $V_{can} = V_{disp}$. 

Now, lets look on the case where a can is not fully submerge and only the part of $h_{sub}$ is submerge. The pressure on the top $P_{top}$ will reduce to $0$ if we are talking about gauge pressure $P_{gauge}$ or equal to atmospheric pressure $P_{atm}$ if we are want the absolute pressure $P_{abs}$. Since above derivation we use the gauge pressure, we can say that the $P_{top} = 0$. This will result to  
\begin{align}
    F_{buoyant} & = P_{bot} A - P_{top} A \quad \quad \text{For the sake of discussion, we say } A_{top} = A_{bot} = A \\
    F_{buoyant} & = P_{bot} A - (0) A \quad \quad \text{The gauge pressure of the top of the can is } 0\\
    F_{buoyant} & = \rho g h_{bot}  \\
    F_{buoyant} & = \rho g A (h_{sub}) \\
    F_{buoyant} & = \rho g A h_{sub} \quad \quad Ah_{sub} \text{ is just the volume submerge or displace }V_{disp} \\
    F_{buoyant} & = \rho g V_{disp} \quad \quad 
\end{align}
We can see here that we derive the buoyancy using the pressure gradient in the object.  <mark> Thus, without a pressure gradient, there will be no buoyant force </mark>. 

Lastly, if we think a case where a liquid pressure are equal at all depths, because of the gauge pressure $P_{gauge} = \rho g h$, this is only happen when the gravity is switch off $g=0$ or the density of the fluid is very near $\rho = 0$. At those cases, the value of $F_{buoyant}$ will also be $0$ since it is also dependent to those variables.